new-site/src/docs/about/resources.mdx

```diff
@@ -23,6 +23,7 @@ import LeadParagraph from '../../components/LeadParagraph';
 - <Link size="M" href="https://saavutettavuusmalli.hel.fi" external> Helsinki Accessibility Model (in Finnish mostly) </Link>  
 - <Link size="M" href="https://pelikirja.hel.fi" external> Helsinki Digital Service Playbook (Pelikirja) (in Finnish) </Link> 
 - <Link size="M" href="https://dev.hel.fi" external> Helsinki Developer guidelines </Link>
+- <Link size="M" href="https://hds.hel.fi/v1" openInNewTab> HDS 1.X documentation </Link>
```

Comment:
I used the term "1.X" at some point but started moving towards "V1" or "version 1". I think it is a bit clearer that way.

Reply:
But whichever seems clearer to you. "1.X" is fine also.